### DISCUSSION TITLE
auth0-client /logout request with client-id

### DISCUSSION THREAD

**Discussion by edocrippaofficial** [mia-platform/community#97] [Ideas]
Hi, I was wondering if the `client-id` can be included in the auth0-client logout request.

From what I can see, now it forwards the request passing only the `returnTo`.
This implies that, at the moment, I have to specify my allowed redirect URLs in the tenant settings and not in the application settings. 

I think it's better to write a return URL specific for an application, to ensure that you can't be redirected to another URL just because you have another application in the same Auth0 tenant.
(And also because I have a lot of applications with a lot of URLs, and it starts to feel a bit messy :))

For reference: https://auth0.com/docs/authenticate/login/logout/redirect-users-after-logout

My only concern is that it could be a breaking-change if someone is writing their allowed URLs in the tenant settings and not in the application settings, so maybe it can be implemented as an option.

**Comment by fredmaggiowski** [comment]
Hi, I agree with you, I think we could implement this and tag a major release. we're already planning it for addressing #78 so I guess this could be included there.

At the beginning we could use an environment variable to control the behaviour and prevent it from being breaking.

If you need the feature you can try and open a PR on the service :)

**Comment by peppemanzi** [comment]
Since the discussion is inactive I close it, after checking that the issue was actually tracked internally.
